EasyRestProjectScreenPlayPattern/Interactions/Tasks/Enter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Boa.Constrictor.Screenplay;
+using Boa.Constrictor.WebDriver;
+using BoaConstrictorTestProject.Components;
+
+namespace BoaConstrictorTestProject.Interactions.Tasks
+{
+    public class Enter : ITask
+    {
+        public static Enter HisPersonalAccount() => new Enter();
+
+        public void PerformAs(IActor actor)
+        {
+            actor.AttemptsTo(Click.On(HeaderMenu.ProfileIcon));
```

Comment:
Do you need to click twice on the Profile Icon?

Reply:
Yes, cause test if flaky without double click